readme.md

```diff
@@ -170,6 +170,7 @@ This readme is auto-generated from the data.js file, so please don't PR this fil
 * [Andrew Byrd](https://www.andrewbyrd.dev/uses/) — Web dev. Girl Dad. Bodybuilder. Gamer.
 * [Siim Männart](https://siim.me/uses) — Cloud Engineer and Developer. Learner of things, recreational pilot and lover of cats. From Tallinn, Estonia. Based in Sydney, Australia.
 * [Prakhil TP](https://www.notion.so/Things-Prakhil-uses-e995e61834c242f1b739be9f8819fb0c) — Team lead, Experienced full-stack engineer & non-stop learner.  :wq
+* [Steve Rydz](https://steverydz.com/uses) — Linux user, Open source, Web developer
```

Comment:
This files gets updated automatically, probably best to remove this change

Reply:
@HugoDF sorry, updated, thanks